src/java.base/windows/native/libjava/WinNTFileSystem_md.c

```diff
@@ -59,7 +59,7 @@ Java_java_io_WinNTFileSystem_initIDs(JNIEnv *env, jclass cls)
 
 /* -- Path operations -- */
 
-extern int wcanonicalize(const WCHAR *path, WCHAR *out, int len);
+extern WCHAR* wcanonicalize(const WCHAR *path, WCHAR *out, int len);
```

Comment:
The first parameter here is `const WCHAR*` but `WCHAR*` in canoncailize_md.c, can you check that?